apps/fsl/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# syntax=docker/dockerfile:1
+FROM ubuntu:24.04
+SHELL ["/bin/bash", "-xe", "-o", "pipefail", "-c"]
+
+ARG APP_NAME
+ARG APP_VERSION
+
+WORKDIR /apps/${APP_NAME}
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update -qy && \
+    apt-get install --no-install-recommends -qy \
+    ca-certificates curl dc file libgomp1 libquadmath0 locales python3 && \
+    locale-gen en_US.UTF-8 en_GB.UTF-8 && \
+    curl -sSL https://fsl.fmrib.ox.ac.uk/fsldownloads/fslinstaller.py -o fslinstaller.py && \
+    sed -i -E "s/(printmsg\(([^,]+, )?end='(\\\\r)?')/# SILENE \\1/g" ./fslinstaller.py && \
+    python3 ./fslinstaller.py \
+        -d /usr/local/fsl \
```

Comment:
Why not `/apps/${APP_NAME}`?

Reply:
hum from memory i'd say that it has some dependencies that are better handled if installed the way it is done there. We can try to install it in /apps/${APP_NAME} but only once i can run some scripts to see that everything works as intended

Reply:
it holds the config files, so I guess it's ok. Ignore my comment.